src/routes/user/usernew.tsx

```diff
@@ -107,6 +107,7 @@ const UserNew: FunctionComponent = () => {
         loadUser();
 
         const saveUser = () => {
+            /*eslint-disable-next-line no-shadow */
             const userInput: userInput = {};
```

Comment:
The type should be pascal-cased (`UserInput`)... I missed that in the review where this was added. :sweat_smile: